### PR TITLE
Add Trey Research

### PR DESCRIPTION
**Pull request by rjfmachado**
This adds the main Readme links to Trey Research.

### CODE REVIEW

**Review by krnese — CHANGES_REQUESTED** [review]

File: README.md [hunk 1]
```diff
@@ -80,6 +81,7 @@ The Enterprise-Scale architecture is modular by design and allows customers to s
...
+| Trey Research | For small and medium Enterprises | [![Deploy To Azure](https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/1-CONTRIBUTION-GUIDE/images/deploytoazure.svg?sanitize=true)](https://portal.azure.com/#blade/Microsoft_Azure_CreateUIDef/CustomDeploymentBlade/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2FEnterprise-Scale%2Fmain%2Fdocs%2Freference%2Ftreyresearch%2FarmTemplates%2Fes-lite.json/createUIDefinitionUri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2FEnterprise-Scale%2Fmain%2Fdocs%2Freference%2Ftreyresearch%2FarmTemplates%2Fportal-es-lite.json) | [Detailed description](./docs/reference/treyresearch/README.md) |
```
[the remark this answers is not in the document]
Comment (rjfmachado) [reply README.md:84]: The readme contains:
This reference implementation provides a design path and initial technical state for Small and Medium Enterprises to start with foundational landing zones that support their application portfolios.

[the remark this answers is not in the document]
Comment (rjfmachado) [reply README.md:84]: we could also use "for small IT teams" ?

[the remark this answers is not in the document]
Comment (krnese) [reply README.md:84]: Can you pls correct so it calls out small enterprise as agreed? 
image

[the remark this answers is not in the document]
Comment (rjfmachado) [reply README.md]: Should be good now @krnese

**Review by krnese — APPROVED** [review]

**Pull request merged**